{
  "cve_id": "CVE-2020-22218",
  "github_data": {
    "repository": "libssh2/libssh2",
    "fix_commit": null,
    "related_commits": [],
    "patch_url": null
  },
  "vulnerability_details": {
    "cvss_score": 7.5,
    "cvss_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "cwe_id": "CWE-787",
    "description": "An issue was discovered in function _libssh2_packet_add in libssh2 1.10.0 allows attackers to access out of bounds memory.",
    "attack_vector": "NETWORK",
    "attack_complexity": "LOW"
  },
  "temporal_data": {
    "published_date": "2023-08-22T19:16:19.120",
    "last_modified": "2024-11-21T05:13:11.797",
    "fix_date": null
  },
  "references": [
    {
      "url": "https://github.com/libssh2/libssh2/pull/476",
      "source": "cve@mitre.org",
      "tags": [
        "Issue Tracking",
        "Patch"
      ]
    },
    {
      "url": "https://lists.debian.org/debian-lts-announce/2023/09/msg00006.html",
      "source": "cve@mitre.org",
      "tags": []
    },
    {
      "url": "https://security.netapp.com/advisory/ntap-20231006-0002/",
      "source": "cve@mitre.org",
      "tags": []
    },
    {
      "url": "https://github.com/libssh2/libssh2/pull/476",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": [
        "Issue Tracking",
        "Patch"
      ]
    },
    {
      "url": "https://lists.debian.org/debian-lts-announce/2023/09/msg00006.html",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": []
    },
    {
      "url": "https://security.netapp.com/advisory/ntap-20231006-0002/",
      "source": "af854a3a-2127-422b-91ae-364da2661108",
      "tags": []
    }
  ],
  "collection_metadata": {
    "collected_at": "2025-01-11T23:06:06.456536",
    "processing_status": "raw"
  }
}